llama_stack/distribution/server/server.py

```diff
@@ -465,6 +466,16 @@ def main(args: argparse.Namespace | None = None):
             window_seconds=window_seconds,
         )
 
+    # --- CORS middleware for local development ---
+    ui_port = os.environ.get("LLAMA_STACK_UI_PORT", 3000)
```

Comment:
- can this port be `8322` by default? 
- can we somehow make it that this is an anonymous port and we have a reverse-proxy in our stack server which sends some paths to this port so there is a single URL which works for the UI? 

we don't need to do these things in this PR, but I think the latter part (a single URL should give me everything especially locally) is important